### DISCUSSION TITLE
Can I send an insecure https request using the $http global object?

### DISCUSSION THREAD

**Discussion by rmoskal** [pocketbase/pocketbase#5553] [Q&A] [answered]
I have an server I need to access that has messed up certificates. I know for example I can do this with the native javascript fetch function by using the rejectUnauthorized  option.

Is there something similar I can use in main.pb.js in a command?

**Comment by ganigeorgiev** [comment]
There is no such option at the moment. I don't understand the use case and I'm not sure if it would be a good idea to add support for it.

**Reply by rmoskal** [reply]
Understiood. The use case is that I have an api I have to call that I have no control over and I need to call it in a pocketbase command to enrich some data in my app.

Most http fetch clients support this.  No worries. My workaround will be to use a proxy.

I was hoping not to.

**Reply by ganigeorgiev** [reply, marked as answer]
Another alternative could be to call an external command like curl using the [$os.cmd helper](https://pocketbase.io/jsvm/functions/_os.cmd.html) but if you already have a proxy it might be the better option.

**Reply by rmoskal** [reply]
The curl thing is probably the better option (not sure its' loaded int the container I'm running).  The proxy is harshing my buzz about pocketbase apps requiring a single artifact.
